database/test/integration/test_pgwar_entity.sql

```diff
@@ -0,0 +1,130 @@
+-- Test the creation of project and community entities
+-- after modifying information.resource and projects.info_proj_rel
+
+-- Start transaction and plan the tests.
+BEGIN;
+SELECT plan(14);
+
+------- Prepare required context data ------
+-- Create and switch to a sink table for entity previews
+SELECT war.create_sink_table_entity_preview('war.e');
```

Comment:
Should the war.e table be used somewhere?

Reply:
This is needed to insert into projects.project (due to a trigger on projects.project).